skrub/_table_vectorizer.py

```diff
@@ -472,6 +495,41 @@ def _clone_transformers(self):
         else:
             self.datetime_transformer_ = self.datetime_transformer
 
+        if (self.specific_transformers is None) or len(self.specific_transformers) == 0:
+            self.specific_transformers_ = []
+        else:
+            first_item_length = len(self.specific_transformers[0])
+            # Check all tuples are the same length
+            for i, tup in enumerate(self.specific_transformers):
+                if len(tup) != first_item_length:
+                    raise TypeError(
+                        "Expected `specific_transformers` to be a list of "
+                        "tuples with all the same length, got length "
+                        f"{len(tup)} at index {i} (elements at previous "
+                        f"indices have {first_item_length} in length). "
+                    )
+            if first_item_length == 2:
+                # Unnamed assignments, transform to named
+                named_specific_transformers = _get_transformer_list(
+                    self.specific_transformers
+                )
+            elif first_item_length == 3:
+                # Named assignments
+                named_specific_transformers = self.specific_transformers
+            else:
+                raise TypeError(
```

Comment:
Can you test this?

Reply:
It is actually already tested [L692](https://github.com/skrub-data/skrub/pull/583/files/586ad0b4d81ef6f1bbaac20b27951b2d0175691f#diff-5dbef96d55da55a37375c0c6cbcd448aa1e5fd189c15c04c9f50b3ecbbb67e02R692), I'm not sure why codecov flags it anyway.

Reply:
Maybe the error got raised elsewhere, could you double-check?

Reply:
Okay, found it, it was a typo on the test, it evaluated the `StandardScaler` instead of passing a tuple.